lib/view_model/anon_invoice_page_view_model.dart

```diff
@@ -182,8 +182,8 @@ abstract class AnonInvoicePageViewModelBase with Store {
 
   @action
   void reset() {
-    selectedCurrency = walletTypeToCryptoCurrency(_wallet.type);
-    cryptoCurrency = walletTypeToCryptoCurrency(_wallet.type);
+    selectedCurrency = walletTypeToCryptoCurrency(_wallet.type, chainId: _wallet.chainId);
+    cryptoCurrency = walletTypeToCryptoCurrency(_wallet.type, chainId: _wallet.chainId);
```

Comment:
sometimes you are using `wallet.currency` and someothers you are using `walletTypeToCryptoCurrency`, let's unify one across all places, so if we will make any modifications it will be in one place